registry/Scripts/self-signed.ps1

```diff
@@ -1,13 +1,27 @@
 #requires -RunAsAdministrator
 
-# Password and Common Name of the certificate
-$PASSWORD=""
-$CN=""
-# Export path of the certificate
-$PATH=""
+Param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "Common DNS FQDN name to be used .")]
+    [string] $DNSCName,
+    [Parameter(Mandatory = $true, HelpMessage = "Certificate Password to be used to export the cert.")]
+    [string] $CertificateSecret,
+    [Parameter(Mandatory = $true, HelpMessage = "Certificate file export path where f.")]
+    [string] $CertificateFileExportPath
+)
+
+if (-not (Test-Path -Path $CertificateFileExportPath -IsValid))
+{
+    Write-Host "Error: Invlid path syntax for CertificateFileExportPath variable."
+}
+
+if (-not ([IO.Path]::GetExtension($CertificateFileExportPath) -eq '.pfx'))
+{
+    Write-Host "Error: Invlid syntax for CertificateFileExportPath variable. Extension value in path should end with '.pfx'"
```

Comment:
typo `Invlid`
is this really a requirement?

Reply:
no just added as I think folks can make mistake.